docs/resources/vcpu_server.md

```diff
@@ -0,0 +1,143 @@
+          target_ip         = ionoscloud_ipblock.example.ips[3]
+          type              = "EGRESS" 
+        }
+    }
+    label {
+        key = "labelkey1"
+        value = "labelvalue1"
+    }
+    label {
+        key = "labelkey2"
+        value = "labelvalue2"
+    }
+}
+resource "random_password" "server_image_password" {
+  length           = 16
+  special          = false
+}
+                  
+```
+
+## Argument reference
+
+- `name` - (Required)[string] The name of the server.
+- `datacenter_id` - (Required)[string] The ID of a Virtual Data Center.
+- `cores` - (Optional)[integer] Number of server CPU cores.
+- `ram` - (Optional)[integer] The amount of memory for the server in MB.
+- `image_name` - (Optional)[string] The name, ID or alias of the image. May also be a snapshot ID. It is required if `licence_type` is not provided. Attribute is immutable.
+- `availability_zone` - (Optional)[string] The availability zone in which the server should exist. E.g: `AUTO`, `ZONE_1`, `ZONE_2`. This property is immutable.
+- `licence_type` - (Optional)[string] Sets the OS type of the server.
+- `volume` - (Required) See the [Volume](volume.md) section.
+- `nic` - (Optional) See the [Nic](nic.md) section.
+- `firewall` - (Optional) Allows to define firewall rules inline in the server. See the [Firewall](firewall.md) section.
+- `boot_volume` - (Computed) The associated boot volume.
+- `boot_cdrom` - (Optional)[string] The associated boot drive, if any.
+- `boot_image` - (Optional)[string] The image or snapshot UUID / name. May also be an image alias. It is required if `licence_type` is not provided.
+- `primary_nic` - (Computed) The associated NIC.
+- `primary_ip` - (Computed) The associated IP address.
+- `firewallrule_id` - (Computed) The associated firewall rule.
+- `firewallrule_ids` - (Computed) The associated firewall rules.
+- `ssh_keys` - (Optional)[list] Immutable List of absolute or relative paths to files containing public SSH key that will be injected into IonosCloud provided Linux images. Also accepts ssh keys directly. Public SSH keys are set on the image as authorized keys for appropriate SSH login to the instance using the corresponding private key. This field may only be set in creation requests. When reading, it always returns null. SSH keys are only supported if a public Linux image is used for the volume creation. Does not support `~` expansion to homedir in the given path.
+- `image_password` - (Optional)[string] The password for the image.
+- `label` - (Optional) A label can be seen as an object with only two required fields: `key` and `value`, both of the `string` type. Please check the example presented above to see how a `label` can be used in the plan. A server can have multiple labels.
+- `inline_volume_ids` - (Computed) A list with the IDs for the volumes that are defined inside the server resource.
+
+> **⚠ WARNING** 
+> 
+> Image_name under volume level is deprecated, please use image_name under server level
```

Comment:
no longer exists I think

Reply:
@cristiGuranIonos done in 35a13f8951cffce1c4c25e74f6242197e6cf8354